packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -296,7 +304,9 @@ storiesOf('ListView/Actions', module)
           </Item>
         </ActionMenu>
       </>
-    ), args));
+    ), args))
+  .add('Restore focus after item removal (disabledBehavior: "selection")', (args) => <FocusExample {...args} />)
+  .add('Restore focus after item removal (disabledBehavior: "all")', (args) => <FocusExample {...args} disabledBehavior="all" />);
```

Comment:
Nit: we can remove this story since we have `disabledBehavior` as a storybook control.